### ISSUE TITLE
translation-injection is checking the wrong thing

### ISSUE THREAD

**Issue by SirPyTech** [OCA/pylint-odoo#571]
Thanks @moylop260 for taking care of https://github.com/OCA/odoo-pre-commit-hooks/pull/192, but it looks like the new PR https://github.com/OCA/pylint-odoo/pull/569 is actually checking something different from https://github.com/OCA/odoo-pre-commit-hooks/pull/192.

Here the check finds occurrences of `_("...".format(...))` like:
https://github.com/OCA/pylint-odoo/blob/ea857599bd3030a50fface17b494e70f9ee5754d/testing/resources/test_repo/broken_module/models/broken_model.py#L506 that are actually fine.

It should find occurrences of `_(...).format()` instead, like https://github.com/OCA/pylint-odoo/blob/ea857599bd3030a50fface17b494e70f9ee5754d/testing/resources/test_repo/broken_module/models/broken_model.py#L514
because this allows translators to leverage the behavior described in https://lucumr.pocoo.org/2016/12/29/careful-with-str-format/ with something like:
```py
_("class of {0}").format(42)
```
Then the translator can translate `"class of {0}"` to `"class of {0.__class__}"` and then Odoo will show `"class of int"`.

The original PR https://github.com/OCA/odoo-pre-commit-hooks/pull/192 was correctly finding https://github.com/OCA/odoo-pre-commit-hooks/blob/35608a626b3358c0857fc8c120568000ac2d1d7c/test_repo/broken_module/models/broken_model.py#L275:
```py
self.message_post(_('Var {a}').format(a=variable1))
```

**Comment by moylop260** [comment]
Agree

Thank you for review it

I will fix it soon
